src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -547,7 +550,7 @@
 								Visibility="{x:Bind ColumnsViewModel.StatusColumn.Visibility, Mode=OneWay}" />
 
 							<controls:GridSplitter
-								Grid.Column="20"
+								Grid.Column="19"
```

Comment:
Does changing the columns affect resizing them?

Reply:
Given that we removed a column, my guess would be no and some testing also indicate resizing still works.

Reply:
Double clicking a grid splitter to resize the column doesn't seem to work. Same with the size all to fit option.

Reply:
Are those behaviors currently present in the store version? I'm not sure what "double clickign" the grid splitter is supposed to do. The behavior for both things you mentioned seems to be the same compared to the store version 🤔

Reply:
Compare with `main`

Reply:
This behavior was broken in the store version and fixed in `main`

Reply:
I think I fixed this now. The whole column logic is a real pain to work with though...

Reply:
You're telling me 😂